src/main/java/com/bynder/sdk/model/Media.java

```diff
@@ -105,6 +105,13 @@
      */
     private Map<String, String> thumbnails;
     /**
+     * Focus point for the current media(image)
+     * The Map contains two keys: 'x' and 'y' for the respective focus point coordinates.
```

Comment:
to make it coherent with the rest of the documentation, write just: "Focus point of the media."